integration-tests/cli/csv-parser-price.it.js

```diff
@@ -103,17 +103,17 @@ describe('CSV and CLI Tests', () => {
       const tmpFile = tmp.fileSync()
       const expectedError = 'Row length does not match headers'
       const csvFilePath = path.join(samplesFolder, 'faulty-sample.csv')
-
-      await expect(
-        exec(
+      try {
```

Comment:
Try/catch was used here to catch the error because async/await could not catch the error due to the introduction of jest-chain dependency.